pkg/services/publicdashboards/api/api.go

```diff
@@ -124,7 +127,7 @@ func (api *Api) SavePublicDashboardConfig(c *models.ReqContext) response.Respons
 	// exit if we don't have a valid dashboardUid
 	dashboardUid := web.Params(c.Req)[":uid"]
 	if dashboardUid == "" || !util.IsValidShortUID(dashboardUid) {
-		handleDashboardErr(http.StatusBadRequest, "no dashboardUid", dashboards.ErrDashboardIdentifierNotSet)
+		api.handleError(http.StatusBadRequest, "No dashboardUid", dashboards.ErrDashboardIdentifierNotSet)
```

Comment:
Errors messages [should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings). What do you think about changing too the older error messages on this package?

Reply:
ah, did not know. Thought we followed sentence casing for everything. I'll change all the error messages to be lowercase 👍